### PR TITLE
[3.x] TinyMCE make use of the core textfilter API for new sites

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

TinyMCE make use of the core textfilter API for new sites

### Testing Instructions

- Install the new package [(from the package step)](https://ci.joomla.org/artifacts/joomla/joomla-cms/staging/28721/downloads/31369/) and check that the default settings for new installs are changed.
- install the update package [(from the package step)](https://ci.joomla.org/artifacts/joomla/joomla-cms/staging/28721/downloads/31369/) and make sure no changes are made for upgrades

### Expected result

The TInyMCE uses the Core textFilter API

### Actual result

The TinyMCE has its own textfilters that just confuses people new to joomla and does not help to protect as everyone is free to change the editor.

### Documentation Changes Required

none

cc @SniperSister  @widmann-it @coolcat-creations

### CODE REVIEW

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on b815f0ec6a933ba6317e67ef2f12af3caae6d766<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28721">issues.joomla.org/tracker/joomla-cms/28721</a>.</sub>

**Comment by Quy** [comment]
It is enabled in Set 2, but not in Set 1, and Set 0.

**Comment by zero-24** [comment]
Please test again @Quy

File: installation/sql/mysql/joomla.sql [hunk 1]
```diff
@@ -586,7 +586,7 @@ INSERT INTO `#__extensions` (`extension_id`, `package_id`, `name`, `type`, `elem
...
...enu":"1","custom_plugin":"","custom_button":""},"1":{"access":["6","2"],"skin":"0","skin_admin":"0","mobile":"0","drag_drop":"1","path":"","entity_encoding":"raw","lang_mode":"1","text_direction":"ltr","content_css":"1","content_css_custom":"","relative_urls":"1","newlines":"0","use_config_textfilters":"1","invalid_elements":"script,applet,iframe","valid_elements":"","extended_elements":"","resizing":"1","resize_horizontal":"1","element_path":"1","wordcount":"1","image_advtab":"0","advlist":"1","autosave":"1","contextmenu":"1","custom_plugin":"","custom_button":""},"0":{"access":["7","4","8"],"skin":"0","skin_admin":"0","mobile":"0","drag_drop":"1","path":"","entity_encoding":"raw","lang_mode":"1","text_direction":"ltr","content_css":"1","content_css_custom":"","relative_urls":"1","newlines":"0","use_config_textfilters":"1","invalid_elements":"script,applet,iframe","valid_elements":"","extended_elements":"","resizing":"1","resize_horizontal":"1","element_path":"1","wordcount":"1","image_advtab":"1","advlist":"1","autosave":"1","contextmenu":"1","custom_plugin":"","custom_button":""}}},"sets_amount":3,"html_height":"550","html_width":"750"}', '', '', 0, '0000-00-00 00:00:00', 3, 0),
```
Comment (Quy) [inline installation/sql/mysql/joomla.sql]: For `invalid_elements`, should the other instances be changed to `""`?

**Comment by zero-24** [comment]
Patched good spot @Quy

**Pull request closed without merging**